engine/standalone/app/src/universal/bin/nussknacker-standalone-entrypoint.sh

```diff
@@ -0,0 +1,31 @@
+#!/usr/bin/env bash
+
+NUSSKNACKER_DIR=`dirname "$0" | xargs -I{} readlink -f {}/..`
+export STORAGE_DIR="$NUSSKNACKER_DIR/storage"
+CONF_DIR="$NUSSKNACKER_DIR/conf"
+LIB_DIR="$NUSSKNACKER_DIR/lib"
+MODELS_DIR="$NUSSKNACKER_DIR/models"
+
+CONFIG_FILE=${NUSSKNACKER_CONFIG_FILE-${2-"$CONF_DIR/application.conf"}}
+LOG_FILE=${NUSSKNACKER_LOG_FILE-${3-"$CONF_DIR/docker-logback.xml"}}
+APPLICATION_APP=${NUSSKNACKER_APPLICATION_APP-${4-"pl.touk.nussknacker.engine.standalone.http.StandaloneHttpApp"}}
+USER=${DAEMON_USER-${5-"daemon"}}
+GROUP=${DAEMON_GROUP-${6-"daemon"}}
+
+mkdir -p ${STORAGE_DIR}/logs
+mkdir -p ${STORAGE_DIR}/db
```

Comment:
standalone doesn't have db as nk. The only db is `/standaloneProcesses` directory